.travis.yml

```diff
@@ -45,6 +46,7 @@ matrix:
     env: USE_CMAKE=YES
 
 script:
+  - if [ ${TRAVIS_OS_NAME} = 'linux' ]; then KWStyle -R -v -gcc -d src; fi
```

Comment:
I could not figure out from the [documentation](https://kitware.github.io/KWStyle/resources/documentation.html) what ``-gcc`` does. What does it do?